src/sage/matroids/matroid.pyx

```diff
@@ -7293,8 +7398,9 @@ cdef class Matroid(SageObject):
 
         INPUT:
 
-        - ``other`` -- a matroid with the same ground set as ``self``.
-        - ``Y`` -- an common independent set of ``self`` and ``other`` of size `k`.
+        - ``other`` -- a matroid with the same groundset as ``self``
+        - ``Y`` -- an common independent set of ``self`` and ``other`` of size
+          `k`
```

Comment:
```suggestion
        - ``Y`` -- an common independent set of ``self`` and ``other`` of size `k`
```
Even if this is slightly over the 80 char/line rule, it looks so much better in the source code and worth the violation.